### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by bqth29**
Remaining unit tests (from #10) :
- [ ] add a test where the time stamp of the task is not in the raoRequest file
- [x] also try create different tests to check the handleRaoRunnerException works as expected with all the different exceptions it can process

### CODE REVIEW

File: gridcapa-core-cc-app/src/test/resources/fileExporterHelper/uploadedNetwork.uct [hunk 1]
```diff
@@ -3,7 +3,7 @@ Generated by powsybl, yyyy-MM-ddTHH:mm:ss.SSS+HH:mm
 Case date: yyyy-MM-ddTHH:mm:ss.SSS+HH:mm
 ##N
 ##ZBE
-BBE1AA1  BE1          0 2 400.00 1000.00 0.00000 0.00000 0.00000 9000.00 -9000.0 9000.00 -9000.0                               F
+BBE1AA1  BE1          0 0        1000.00 0.00000 0.00000 0.00000
```
Comment (kahyami) [inline gridcapa-core-cc-app/src/test/resources/fileExporterHelper/uploadedNetwork.uct]: why these values ​​changed ?

File: gridcapa-core-cc-app/src/test/resources/util/TestCase12NodesHvdc.uct [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ##C 2007.05.01
 ##N
 ##ZBE
-BBE1AA1  BE1          0 2 400.00 1000.00 0.00000 0.00000 0.00000 9000.00 -9000.0 9000.00 -9000.0
+BBE1AA1  BE1          0 0 400.00 1000.00 0.00000 0.00000 0.00000 0000.00 00000.0 0000.00 00000.0
```
Comment (kahyami) [inline gridcapa-core-cc-app/src/test/resources/util/TestCase12NodesHvdc.uct]: why these values ​​changed ?

File: gridcapa-core-cc-app/src/test/resources/fileExporterHelper/uploadedNetwork.uct (hunk 1 above)
Comment (danielthirion) [reply gridcapa-core-cc-app/src/test/resources/fileExporterHelper/uploadedNetwork.uct]: deleted in a new commit

File: gridcapa-core-cc-app/src/test/resources/util/TestCase12NodesHvdc.uct (hunk 2 above)
Comment (danielthirion) [reply gridcapa-core-cc-app/src/test/resources/util/TestCase12NodesHvdc.uct]: deleted in a new commit

**Review by kahyami — APPROVED** [review]

**Pull request merged**